docs/en/docs/kangaroo/intro/release-notes.md

```diff
@@ -12,38 +12,50 @@ understand the evolution path and feature changes of each version.
 *[Kpanda]: Internal dev codename for container management
 *[Harbor]: An open-source container registry tool and a CNCF graduated project
 
-## 2023-07-31
+## 2024-08-30
+
+### v0.21.1
+
+- **Added** a feature of quickly creating managed Harbor instances through a one-click setup for Redis/Minio/PG.
+- **Optimized** the validation of middleware instances when creating managed Harbor and disallowed reuse.
+- **Optimized** the middleware module so that middleware created through the container registry can also be managed by the middleware module.
```

Comment:
```suggestion
- **Optimized** the adaptability to middlewares,  so that a middleware created through the container registry can also be managed by the DCE 5.0 middleware (`mcamel`) module.
```